<center>
    <img src="https://cf-courses-data.s3.us.cloud-object-storage.appdomain.cloud/IBMDeveloperSkillsNetwork-DS0105EN-SkillsNetwork/labs/Module2/images/SN_web_lightmode.png" width="300" alt="cognitiveclass.ai logo"  />
</center>


#### Add your code below following the instructions given in the course


<h1>My Jupyter Notebook on IBM Watson Studio</h1>

<b>Ken Tang</b>

<p>IT Support <del>Dog</del> Engineer</p>

<em>I know MySQL and Python, which type of role can I do?</em>

<h3>You are going to buy some clothing, how much you should pay?</h3>

In [2]:
x = [99,149,79,59,199,399]
y = ["Anya T-Shirt", "Hoodie", "L/S T-Shirt", "S/S T-shirt", "Jeans", "Business Jacket"]
print("It would cost you $", sum(x), "in total")
print("With this receipt and tag, you can exchange or refund within 14 days!")


It would cost you $ 984 in total
With this receipt and tag, you can exchange or refund within 14 days!


| Item          | Price         |
| ------------- |:-------------:|
| Anya T-Shirt  | 99            |
| Hoodie        | 149           |
| L/S T-Shirt   | 79            |
| S/S T-Shirt   | 59            |
| Jeans         | 199           |
|Business Jacket| 399           |
| **Total**     | **984**       |

https://www.uniqlo.com.hk/
